modules/n1ql/pages/n1ql-language-reference/searchfun.adoc

```diff
@@ -310,33 +310,33 @@ This query returns 5 results, and the results are ordered, as specified by the s
 As an alternative, you could limit the number of results and order them using the N1QL xref:n1ql-language-reference/limit.adoc[LIMIT] and xref:n1ql-language-reference/orderby.adoc[ORDER BY] clauses.
 ====
 
-.Search against a full text search index that carries a custom type mapping
+.Search against a full text search index that carries a custom type mapping on documents with "type" field: "hotel"
 ====
 [source,n1ql]
 ----
 SELECT META(t1).id
 FROM `travel-sample` AS t1
-WHERE t1.type = "airline" AND SEARCH(t1.country, "United States");
+WHERE t1.type = "hotel" AND SEARCH(t1.country, "United States");
 ----
 
 .Results
 [source,json]
 ----
 [
   {
-    "id": "airline_2577"
+    "id": "hotel_3426"
   },
   {
-    "id": "airline_2395"
+    "id": "hotel_25504"
   },
   {
-    "id": "airline_2657"
+    "id": "hotel_37597"
   },
 ...
 ]
 ----
 
-If the full text search index being queried has its default mapping disabled and has a custom type mapping defined ("airline" in the above example), the query needs to specify the type explicitly.
+If the full text search index being queried has its default mapping disabled and has a custom type mapping defined ("hotel" in the above example), the query needs to specify the type explicitly. For a sample index definition for the above query, refer to xref:fts-demonstration-indexes.adoc#travel-sample-index-hotel-description-country[Travel sample index definition over documents of type: "hotel"].
```

Comment:
```suggestion
If the full text search index being queried has its default mapping disabled and has a custom type mapping defined ("hotel" in the above example), the query needs to specify the type explicitly.
For a sample index definition for the above query, refer to xref:fts-demonstration-indexes.adoc#travel-sample-index-hotel-country-description[Travel sample index definition over documents of type: "hotel"].
```

Reply:
New line for new sentence, and section ID did not match the section ID in `fts-demonstration-indexes.adoc`.

Reply:
Done.